README.md

```diff
@@ -1,6 +1,58 @@
-# cluster-autoscaler-operator
+# Cluster Autoscaler Operator
 
-The cluster-autoscaler-operator manages Kubernetes
-[cluster-autoscaler][cluster-autoscaler] deployments.
+The cluster-autoscaler-operator manages deployments of the OpenShift
+[Cluster Autoscaler][1] using the [cluster-api][2] provider.
 
-[cluster-autoscaler]: https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
+[1]: https://github.com/openshift/kubernetes-autoscaler/tree/master/cluster-autoscaler
+[2]: https://github.com/kubernetes-sigs/cluster-api
+
+
+## Custom Resource Definitions
+
+The operator manages the following custom resources:
+
+- __ClusterAutoscaler__: This is a singleton resource which controls the
+  configuration of the cluster's autoscaler instance.  The operator will
+  only respond to the ClusterAutoscaler resource named "default" in the
+  managed namespace, i.e. the value of the `WATCH_NAMESPACE` environment
+  variable.  ([Example][ClusterAutoscaler])
```

Comment:
Can you describe how are individual fields of `.spec` translated into the cluster autoscaler options? So readers can mentally connect the CRD example with the CA flags?

Reply:
I struggled with how to put this into the text in a way that was readable. I ended up just showing the arguments from the pod that you get if you apply the example linked. Let me know if you think that's enough. Or maybe we could put that as a comment in the example itself.

Reply:
Good enough for me!!! Thanks